CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 cookbook-rb-manager CHANGELOG
 ===============
 
+## 2.7.2
+
+  - Luis Blanco
+    - [bf8d086] Merge remote-tracking branch 'origin/master' into feature/#16519_fix_ale_action_logic
```

Comment:
Lines of merges can be safely discarded from the changelog since they are not significant changes fot the task